.gitignore

```diff
@@ -63,3 +63,4 @@ src/mkdocs-codeinclude-plugin
 src/pip-delete-this-directory.txt
 
 .DS_Store
+.directory
```

Comment:
I'm not against adding things to `.gitignore`, but it looks like this might have been added accidentally.

Reply:
That was not any accident. My OS generates this file, when the directory should show dot-files in the UI ;).

Reply:
I've never seen `.directory` being generated before. What's your OS? 😄 

If this is something that's fairly unique to your development environment, perhaps this belongs in your global gitignore rather than in individual repos.

Reply:
I am working with Dolphin on `Ubuntu 18.04.2 LTS`.
I will remove it, due to making global settings of it. Did some research.